<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="graphicsGB">
     <property name="title">
      <string>Graphics</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="scrSizeLbl">
        <property name="text">
         <string>Screen resolution:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="scrSizeAuto">
          <property name="text">
           <string>Autodetect</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="width">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>640</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="xLbl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="height">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>480</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Video menory size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
        <item>
         <widget class="QCheckBox" name="vmAuto">
          <property name="text">
           <string>Autodetect</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="vm">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>32</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mbLbl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">MB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useScripts">
     <property name="text">
      <string>Use additional scripts(WARNING: unsafe option!)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrSizeAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>width</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrSizeAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>height</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrSizeAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>xLbl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>vm</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>mbLbl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
